File: app/src/main/java/com/example/letmebeyourchef/Listeners/ResponseListenerRicetteRandom.java
package com.example.letmebeyourchef.Listeners;

import com.example.letmebeyourchef.Models.ResponseFromApiRicetteRandom;

public interface ResponseListenerRicetteRandom {
    void didFetch(ResponseFromApiRicetteRandom response, String message);
    void didError(String message);
}

File: app/src/main/java/com/example/letmebeyourchef/Listeners/ResponseListenerRicetteRandom.kt
package com.example.letmebeyourchef.Listeners

import com.example.letmebeyourchef.Models.ResponseFromApiRicetteRandom

open interface ResponseListenerRicetteRandom {
    fun didFetch(response: ResponseFromApiRicetteRandom?, message: String?)
    fun didError(message: String?)
}